README.md

```diff
@@ -671,6 +671,7 @@ Name | Website | Region
 [Udacity](/company-profiles/udacity.md) | https://www.udacity.com/ | Worldwide
 [Uhuru](/company-profiles/uhuru.md) | https://uhurunetwork.com/ | Worldwide
 [Uncapped](/company-profiles/uncapped.md) | https://weareuncapped.com/ | Europe, USA
+[Uplers](/company-profiles/uplers.md) | https://www.uplers.com/ | Worldwide
```

Comment:
Although you may hire globally (?), 

- You wrote India within the region of the profile,
- The user is greeted with `Opening Global Doors for INDIA’S TOP TALENT` when they open the website 
- Your office locations are just listed at `India`
- and finally you state `with the best remote Indian talent to meet their hiring needs.` within the blurb.

For the above reasons, could you change `Worldwide` to `India`

Reply:
requested information provided